### PR TITLE
Fix style guide mismatches

### PR DESCRIPTION
**Pull request by andrecardoso**
## Description
Fix defects related to style guide mismatching: 

- https://github.com/Rise-Vision/rise-vision-apps/issues/1068
- https://github.com/Rise-Vision/rise-vision-apps/issues/1069
- https://github.com/Rise-Vision/rise-vision-apps/issues/1070
- Everything in [this document](https://docs.google.com/document/d/13hSHSo8FMMo9CXDo4lutGRXcLCwCTGefOC-EXPTMhog/edit?ts=5d3634cb) except modals 

## Motivation and Context

Current UI doesn't match with the style guide

## How Has This Been Tested?

Changes are deployed to [stage-3](https://apps-stage-3.risevision.com/editor/list?cid=b258739a-527b-438c-b6ec-01057cecdfa2). I tested on both desktop and mobile using browserstack.

## Release Plan:
**Automated Test** No need to update, as it was only a style change
**Monitoring** Will validate the release once in Production
**Rollback** Rollback can be done by reverting the PR and following the deployment process
**Documentation**  No need to update documentation
**Internal Communication** I will post a message on slack regarding these changes

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

Related common-header PR: https://github.com/Rise-Vision/common-header/pull/1023

### CODE REVIEW

**Comment by andrecardoso** [comment]
@sheadarlison these changes are all UI related, could you please review it in stage-3? https://apps-stage-3.risevision.com/editor/list

Thanks.

**Comment by andrecardoso** [comment]
[(3) [Issue #1068] rise-data-financial does not follow the style guide](https://trello.com/c/pbsf4QaQ/184-3-issue-1068-rise-data-financial-does-not-follow-the-style-guide)

**Comment by andrecardoso** [comment]
[(3) [Issue #1069] Template editor header does not match style guide](https://trello.com/c/jKZK8Eli/185-3-issue-1069-template-editor-header-does-not-match-style-guide)

**Comment by andrecardoso** [comment]
[(3) [Issue #1070] Component list in template editor does not match style guide](https://trello.com/c/rFHHVsTv/191-3-issue-1070-component-list-in-template-editor-does-not-match-style-guide)

File: web/partials/template-editor/component.html [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-                    icon="{{ panelIcon }}"
-                    type="{{ panelIconType }}">
-    </component-icon>
-    <span ng-show="!panelTitle">
-      {{ ( "template." + factory.selected.type ) | translate }} -
-      {{ factory.selected.label }}
+<div class="component-header" ng-class="{ 'visible-bottom-rule': isHeaderBottomRuleVisible(factory.selected) }">
+  <a ng-click="onBackButton();">
+    <span class="attribute-edit text-nowrap">
+      <i class="fa fa-lg fa-angle-left arrow-icon"></i>
```
Comment (olegrise) [inline web/partials/template-editor/component.html]: @andrecardoso should we be using streamlineicons here or you waiting for Jason?

Comment (andrecardoso) [reply web/partials/template-editor/component.html]: Waiting for Jason, I couldn't find the icon file.

**Comment by olegrise** [comment]
@andrecardoso I reviewed. Looks good. The only outstanding question is about replacing font Awesome icon with streamline icon.

**Comment by sheadarlison** [comment]
@andrecardoso could you please have @jasoncaldwell review this instead of myself. Thanks

**Comment by andrecardoso** [comment]
[(2) Merge/deploy UI style guide fixes](https://trello.com/c/nZ6KTJ2z/193-2-merge-deploy-ui-style-guide-fixes)

**Comment by jasoncaldwell** [comment]
@andrecardoso Visual defects in this PR:

***

### Label Text Size

Rise_Vision___Apps

Text labels need to be 14px. Right now it's 12px.

***

### Green Check Mark

Rise_Vision___Apps

Check Mark icon shouldn't be green when the box is empty. Please refer to [the style guide](https://docs.google.com/document/d/1JaurbTMAWsHZIauyrENMVdM1shwdzb5BqtKkjSURVoI/edit#bookmark=id.v5boyiuwkyio).

***

### "Publish To Display" button style, on hover

Rise_Vision___Apps

"Publish To Display" button shouldn't change color or style on hover.

**Comment by andrecardoso** [comment]
@jasoncaldwell I fixed the problems you listed. Please re-review. Thanks!

**Comment by jasoncaldwell** [comment]
@andrecardoso 

***
### Text labels need more margin underneath

Rise_Vision___Apps

There needs to be 5px of additional margin underneath the text labels.

**Comment by andrecardoso** [comment]
@jasoncaldwell fixed. Please re-review. Thanks

**Comment by jasoncaldwell** [comment]
@andrecardoso Looks good. 👍

**Pull request merged**